modules/seqexec/engine/src/test/scala/seqexec/engine/StepSpec.scala

```diff
@@ -382,8 +394,8 @@ class StepSpec extends FlatSpec {
       case Some(Sequence.State.Zipper(zipper, status, _)) =>
         inside (zipper.focus.toStep) {
           // Check that the sequence stopped midway
-          case Step(_, _, _, _, ex1::ex2::ex3::Nil) =>
-            assert( Execution(ex1).results.length == 2 && Execution(ex2).results.length == 1 && Execution(ex3).actions.length == 1)
+          case Step(_, _, _, _, ex1 :: ex2 :: ex3 :: Nil) =>
```

Comment:
You changed the rest of them from `ex1 :: ex2 :: ... :: Nil` to `List(ex1, ex2, ...)`. Why not this one?

Reply:
No reason really, I should use the same style